### ISSUE TITLE
adjective moves before determiner

### ISSUE THREAD

**Issue by jeisner** [arendu/MacaronicParseUI#15]
In the demo, the 4th sentence translates "les avocats républicains" to "Republican the lawyers" instead of "the Republican lawyers."  Just a bad alignment, or is something worse going on?

**Comment by arendu-zz** [comment]
Yes, les is originally aligned to null. My script forces it to align to  avocats. To ensure alignments are symmetric
